src/dc_tools.rs

```diff
@@ -330,13 +330,13 @@ fn encode_66bits_as_base64(v1: u32, v2: u32, fill: u32) -> String {
     let mut wrapped_writer = Vec::new();
     {
         let mut enc = base64::write::EncoderWriter::new(&mut wrapped_writer, base64::URL_SAFE);
-        enc.write_u32::<BigEndian>(v1).unwrap();
-        enc.write_u32::<BigEndian>(v2).unwrap();
-        enc.write_u8(((fill & 0x3) as u8) << 6).unwrap();
-        enc.finish().unwrap();
+        enc.write_u32::<BigEndian>(v1).unwrap_or_default();
+        enc.write_u32::<BigEndian>(v2).unwrap_or_default();
+        enc.write_u8(((fill & 0x3) as u8) << 6).unwrap_or_default();
+        enc.finish().unwrap_or_default();
```

Comment:
Here's what I understand: the base64 EncodedWriter promises to not create errors, only passed through errors from the wrapped writer: https://docs.rs/base64/0.10.1/base64/write/struct.EncoderWriter.html?search=#errors

And the wrapper writer is a Vec which will always return Ok: https://doc.rust-lang.org/src/std/io/impls.rs.html#313-316

Therefore I think plain .unwrap() here is correct.

I must admit I don't follow where write_u32 comes from but I assume this is very similar.  /cc @link2xt who wrote this